doc/examples/bundle_extraction.py

```diff
@@ -155,10 +155,11 @@
 ren.SetBackground(1, 1, 1)
 ren.add(actor.line(model_bundle, colors=(.1,.7,.26)))
 ren.add(actor.line(recognized_bundle, colors=(.1,.1,6)))
+ren.set_camera(focal_point=(-18.17281532, -19.55606842, 6.92485857),
+               position=(-360.11, -340.46, -40.44),
+               view_up=(-0.03, 0.028, 0.89))
 window.record(ren, out_path='CST_L_recognized_bundle.png',
-              ren.set_camera(focal_point=(-18.17281532, -19.55606842, 6.92485857),
-                             position=(-360.11, -340.46, -40.44),
-                             view_up=(-0.03, 0.028, 0.89)), size=(600, 600))
+              size=(600, 600))
 if interactive:
     window.show(ren)
```

Comment:
I believe in show and record you have to say reset_camera=False

Reply:
@BramshQamar let me know if this is not necessary.

Reply:
It works fine without setting reset_camera=False.
if I set it to False then our streamlines appear to be very far and zoomed out.

Reply:
I believe still fury updates the camera automatically when reset_camera=True. Only for scaling though.

Reply:
if I do set_camera=False

this is how it saves the .png

![af_l_recognized_bundle](https://user-images.githubusercontent.com/26060471/49691253-11b42c80-fb0c-11e8-8edd-f5e7936421e7.png)

Reply:
we cannot see anything

Reply:
I can't see anything in there? :smile:

Seems like maybe a bug in fury?